src/Extension/SandboxExtension.php

```diff
@@ -21,16 +21,13 @@
 
 final class SandboxExtension extends AbstractExtension
 {
-    private $sandboxedGlobally;
-    private $sandboxed;
-    private $policy;
-    private $sourcePolicy;
 
-    public function __construct(SecurityPolicyInterface $policy, $sandboxed = false, ?SourcePolicyInterface $sourcePolicy = null)
+    public function __construct(
+        private SecurityPolicyInterface $policy,
+        private bool $sandboxed = false,
+        private ?SourcePolicyInterface $sourcePolicy = null,
+    )
     {
```

Comment:
CS fix for all such changes:

```suggestion
    ) {
```